### DISCUSSION TITLE
Portfolio Ideas - An open-source repository for inspiration

### DISCUSSION THREAD

**Discussion by giscus** [Evavic44/victoreke.com#100] [Announcements]
# Portfolio Ideas - An open-source repository for inspiration

Are you embarking on the journey of creating your own portfolio site but lack design inspiration? Explore portfolio ideas, a rich curated collection of developer portfolios that will spark your imagination

https://victoreke.com/blog/portfolio-ideas-an-open-source-repository-for-inspiration

**Comment by odhyp** [comment]
Cool project!

**Reply by Evavic44** [reply]
Thanks :)

**Comment by denverdelamasa** [comment]
everybody's works are CRAZYY!!!!! T_T_T  I'm very inspired by you guys!!! :DDDD
